doc/source/whatsnew/v0.24.0.txt

```diff
@@ -573,6 +573,5 @@ Other
 - :meth: `~pandas.io.formats.style.Styler.background_gradient` now takes a ``text_color_threshold`` parameter to automatically lighten the text color based on the luminance of the background color. This improves readability with dark background colors without the need to limit the background colormap range. (:issue:`21258`)
 - Require at least 0.28.2 version of ``cython`` to support read-only memoryviews (:issue:`21688`)
 - :meth: `~pandas.io.formats.style.Styler.background_gradient` now also supports tablewise application (in addition to rowwise and columnwise) with ``axis=None`` (:issue:`15204`)
--
--
+- :meth:`pandas.core.algorithms.isin` avoids spurious casting for lists (:issue:`22160`)
```

Comment:
is this user visible?

Reply:
@jreback  Only when the user uses `pandas.core.algorithms.isin` directly, then the wrong behavior from #22160 is fixed. There is however no difference if `isin` is used via `Series` or `Index` - the values are already in a `np.array` and thus the bug ins't triggered.

Reply:
ok , this is an internal, routine, ok removing this whatsnew note.